common/workunit/workunit.cpp

```diff
@@ -6767,7 +6767,15 @@ void CLocalWorkUnit::createGraph(const char * name, const char *label, WUGraphTy
 
 IConstWUGraphProgress *CLocalWorkUnit::getGraphProgress(const char *name) const
 {
-    throwUnexpected();   // Should only be used for persisted workunits
+/*    Owned<IRemoteConnection> conn = getProgressConnection();
```

Comment:
Not sure really what it should do - this is used when outputting WU to XML, so returning NULL just means that any graph progress that is created in the local WU will not be output.... but there should possibly be a change in the code that writes progress TO the local WU.